src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -626,6 +744,12 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                                                       INS_SCALABLE_OPTS_UNPREDICATED);
                     }
                     break;
+                case 4:
```

Comment:
What code is hitting this path?

Reply:
Is it the `*BySelectedIndex` variants?

Reply:
I think I added it here, but forgot to remove it when I moved the code above. Will remove it.

Reply:
👍, was asking since the assert is saying it shouldn't be RMW, but all the FMA instructions are